src/zope/errorview/browser.py

```diff
@@ -52,6 +52,8 @@ def update(self):
             'Cache-Control', 'no-store, no-cache, must-revalidate')
         self.request.response.setHeader(
             'Pragma', 'no-cache')
+        self.request.response.setHeader(
+            'Content-Type', 'text/plain;charset=utf-8')
```

Comment:
This doesn't seem to be related to Python 3 changes?

Reply:
This relates to one of your general remarks above.

These changes are in fact related to Python 3. Since in Python 2 when returning "just" a string as a response would have `zope.publisher` not require a content-type to be set as the response body would not be unicode.

In Python 3 "just" returning a string as response obviously *is* unicode and `zope.publisher` then requires (otherwiser https://github.com/zopefoundation/zope.publisher/blob/c1ba72f6a3838269badc433ba9f06a4f1a0eabf2/src/zope/publisher/http.py#L821 would fail) a content type to be set. In other words, it became more important than before to make sure you set the correct content type in your application for your views - which in itself is a good thing I think.

I'll make a note of this in CHANGES.rst indeed and bump the version to 1.1.

Reply:
Thank you for explaining!